server/src/db/migrations/20220201192529_createReviews.cjs

```diff
@@ -10,11 +10,8 @@ exports.up = async (knex) => {
     table.bigIncrements("id");
     table.integer("rating").notNullable();
     table.string("description");
-    table.integer("upvotes");
-    table.integer("downvotes");
```

Comment:
Careful making changes to existing migrations that have been merged in! This change will need to be communicated to the group to let everyone know that the database schema will now be altered going forward if this is merged in. This change would be affecting the existing migrations on everyone else's machine. When making database changes, we should ideally do so in new migrations so that there is a clear history of changes against the database.

Reply:
This was a group decision to delete straight off of the migration file. As we are correcting something that shouldn't have been there, not adding something in. we thought it was more of a system design issue.